apps/opencs/view/render/scenewidget.cpp

```diff
@@ -35,14 +39,33 @@ namespace CSVRender
 {
 
 RenderWidget::RenderWidget(QWidget *parent, Qt::WindowFlags f)
-    : QWidget(parent, f)
+    : QOpenGLWidget(parent, f)
     , mRootNode(0)
 {
-
-    osgViewer::CompositeViewer& viewer = CompositeViewer::get();
-
     osg::DisplaySettings* ds = osg::DisplaySettings::instance().get();
-    //ds->setNumMultiSamples(8);
+    ds->setNvOptimusEnablement(1);
+    ds->setStereo(false);
+    ds->setNumMultiSamples(8);
```

Comment:
Either leave this untouched or figure out if we've got a setting for it in `openmw-cs.cfg`.